includes/class-wc-admin-install.php

```diff
@@ -189,6 +189,11 @@ private static function get_schema() {
 			UNIQUE KEY user_id (user_id),
 			KEY email (email)
 		) $collate;
+		CREATE TABLE {$wpdb->prefix}wc_product_category_lookup (
+			product_id BIGINT UNSIGNED NOT NULL,
```

Comment:
Getting depth might be tricky as the hierarchy WP stores is flat. Doing this would require a query for each and every term in the store which won't scale.

We might be able to do what we need without depth. It won't be as useful for all use cases, but it will allow quick lookup of all decendants.

Reply:
@joshuatf As I feared, depth is too much of a performance bottleneck -.- It requires looping over every term, and I also found that if you update a term you need to update ALL parents and ALL descendants because depth may change.

I'll scrapping depth and just storing hierarchy instead. That is working and it's more efficient.